custom/icds/form_processor/steps.py

```diff
@@ -0,0 +1,15 @@
+from corehq.form_processor.steps import VaultPatternExtractor
+
+
+AADHAAR_XFORM_SUBMISSION_PATTERNS = [r'<aadhar_number>(\d{12})<\/aadhar_number>']
```

Comment:
could use a test case

Reply:
`TestVaultPatternExtractor` is built on same lines. You mean using the same case property name?

Reply:
yea, I mean just test the regex

Reply:
https://github.com/dimagi/commcare-hq/pull/28161/commits/0b631635aafbbfb7551b8278e32022dbddce46fd